process_h5s.py

```diff
@@ -1,28 +1,45 @@
 #!/usr/bin/env python3
 
+import sys
 import argparse
 import csv
 import logging
 import pathlib
 import json
+import yaml
 import datetime
 
 import numpy as np
 import pandas as pd
 
+
 from epifor.gleam.simulation import SimSet, Simulation
 from epifor.regions import Regions
 from epifor.data.export import ExportDoc, ExportRegion
 
 log = logging.getLogger("process_data_hdf")
 
 
-def make_export_doc(regions, simset):
+def match_params_with_config(seasonality, air_traffic, config):
+    for conf in config['scenarios']:
+        if (
+                conf['param_seasonalityAlphaMin'] == seasonality and
+                conf['param_occupancyRate'] == air_traffic
+        ):
+            lname = conf['name']
+            return lname
+
+    raise Exception("None Seasonality and air_traffic found")
+
+
+def make_export_doc(regions, simset, config):
```

Comment:
Btw. this seems like an unused argument, no? (I know it's here from the previous time)